test/DebugUrlTrackerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+namespace Hostnet\HnDependencyInjectionPlugin;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+
+/**
+ * @covers \Hostnet\HnDependencyInjectionPlugin\DebugUrlTracker
+ */
+class DebugUrlTrackerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var DebugUrlTracker
+     */
+    private $debug_url_tracker;
+
+    /**
+     * @dataProvider dataProvider
+     */
+    public function testOnKernelResponse($headers, $hasXDebugTokenLink, $isMasterRequest, $expectDebugBar)
+    {
+
+        $symfony1_response = $this->prophesize(\sfWebResponse::class);
+        //Use supplied Content-Type for sf1 Context
+        $symfony1_response->getContentType()->willReturn($headers['Content-Type']);
+
+        $symfony1_context = $this->prophesize(Symfony1Context::class);
+        $symfony1_context->isInitialized()->willReturn(true);
+        $symfony1_context->getResponse()->willReturn($symfony1_response->reveal());
+
+        $this->debug_url_tracker = new DebugUrlTracker($symfony1_context->reveal());
```

Comment:
I would just use it as $debug_url_tracker, no need to declare it as a class property.